README
NAME
    Task::PYX - Install the PYX modules.

SYNOPSIS
     cpanm Task::PYX

SEE ALSO
    App::PYX2XML
        Perl class for pyx2xml application.

    PYX A perl module for PYX handling.

    PYX::GraphViz
        GraphViz output for PYX handling.

    PYX::Hist
        Processing PYX data or file and print histogram.

    PYX::Parser
        PYX parser.

    PYX::SGML::Raw
        Processing PYX data or file and write as SGML.

    PYX::SGML::Tags
        Processing PYX data or file and write as SGML via Tags.

    PYX::Sort
        Processing PYX data or file and sort element attributes.

    PYX::Stack
        Processing PYX data or file and process element stack.

    PYX::Utils
        PYX common utilities for encoding and decoding data.

    PYX::XMLNorm
        Processing PYX data or file and do XML normalization.

    PYX::XMLSchema::List
        Processing PYX data or file and print list of XML schemas.

    SGML::PYX
        Convertor between SGML and PYX.

    Tags::Output::PYX
        Tags output to PYX format.

    XML::MinWriter
        Perl extension for writing XML in PYX format.

    XML::PYX
        PYX parser based on XML::Parser.

REPOSITORY
    <https://github.com/michal-josef-spacek/Task-PYX>

AUTHOR
    Michal Josef Špaček <mailto:skim@cpan.org>

    <http://skim.cz>

LICENSE AND COPYRIGHT
    © 2015-2023 Michal Josef Špaček

    Artistic License

    BSD 2-Clause License

VERSION
    0.11



Changes
0.11

0.10 2023-12-01T15:02:20+01:00
 - Add SGML::PYX 0.02 version.
 - Fix use lib '.' in Makefile.PL.
 - Improve LICENSE AND COPYRIGHT section in doc.
 - Move bugtracker to github.
 - Update App::PYX2XML to 0.06 version.
 - Update Module::Install to 1.21 version.
 - Update PYX to 0.10 version.
 - Update PYX::GraphViz to 0.06 version.
 - Update PYX::Hist to 0.08 version.
 - Update PYX::SGML::Raw to 0.05 version.
 - Update PYX::SGML::Tags to 0.10 version.
 - Update PYX::Sort to 0.05 version.
 - Update PYX::Stack to 0.06 version.
 - Update PYX::Utils to 0.07 version.
 - Update PYX::XMLNorm to 0.05 version.
 - Update PYX::XMLSchema::List to 0.06 version.
 - Update SGML::PYX to 0.07 version.
 - Update Tags::Output::PYX to 0.05 version.
 - Update author github username.
 - Update author name.
 - Update copyright years.

0.09 2016-02-17T12:02:11+01:00
 - Add App::PYX2XML 0.01 version.
 - Add PYX::SGML::Raw 0.01 version.
 - Update PYX::XMLSchema::List to 0.04 version.
 - Update copyright years.

0.08 2015-10-25T17:42:21+01:00
 - Fix PYX description in SEE ALSO section.
 - Move author tests to xt/ directory.
 - Update Module::Install to 1.16 version.
 - Update PYX to 0.04 version.
 - Update PYX::GraphViz to 0.04 version.
 - Update PYX::Hist to 0.03 version.
 - Update PYX::SGML::Raw to 0.02 version.
 - Update PYX::Sort to 0.03 version.
 - Update PYX::Stack to 0.04 version.
 - Update PYX::Utils to 0.04 version.
 - Update PYX::XMLNorm to 0.04 version.
 - Update PYX::XMLSchema::List to 0.03 version.
 - Update Tags::Output::PYX to 0.03 version.

0.07 2015-04-05T00:37:18+02:00
 - Add PYX::SGML::Raw 0.01 version.
 - Add PYX::Sort to SEE ALSO section in doc.
 - Add PYX::XMLSchema::List 0.01 version.

0.06 2015-03-31T23:18:34+02:00
 - Add XML::PYX 0.07 version.
 - Add XML::MinWriter 0.08 version.
 - Rewrite SEE ALSO section to NEILB format.
 - Update PYX::XMLNorm to 0.03 version.

0.05 2015-03-22T01:16:07+01:00
 - Add PYX::Sort 0.02 version.
 - Add PYX::XMLNorm 0.01 version.
 - Add Tags::Output::PYX 0.01 version.
 - Update PYX to 0.03 version.

0.04 2015-02-04T23:10:02+01:00
 - Update PYX::Hist to 0.02 version.
 - Update PYX::Stack to 0.03 version.
 - Update PYX::Utils to 0.03 version.

0.03 2015-01-26T01:00:34+01:00
 - Add PYX::Hist 0.01 version.
 - Add missing PYX::Parser to SEE ALSO section in doc.
 - Update PYX::Stack to 0.02 version.

0.02 2015-01-18T19:00:08+01:00
 - Add PYX::Stack 0.01 version.
 - Update PYX to 0.02 version.
 - Update PYX::GraphViz to 0.03 version.

0.01 2015-01-16T17:40:44+01:00
 - First version


PYX.pm
package Task::PYX;

use strict;
use warnings;

our $VERSION = 0.11;

1;

__END__

=pod

=encoding utf8

=head1 NAME

Task::PYX - Install the PYX modules.

=head1 SYNOPSIS

 cpanm Task::PYX

=head1 SEE ALSO

=over

=item L<App::PYX2XML>

Perl class for pyx2xml application.

=item L<PYX>

A perl module for PYX handling.

=item L<PYX::GraphViz>

GraphViz output for PYX handling.

=item L<PYX::Hist>

Processing PYX data or file and print histogram.

=item L<PYX::Parser>

PYX parser.

=item L<PYX::SGML::Raw>

Processing PYX data or file and write as SGML.

=item L<PYX::SGML::Tags>

Processing PYX data or file and write as SGML via Tags.

=item L<PYX::Sort>

Processing PYX data or file and sort element attributes.

=item L<PYX::Stack>

Processing PYX data or file and process element stack.

=item L<PYX::Utils>

PYX common utilities for encoding and decoding data.

=item L<PYX::XMLNorm>

Processing PYX data or file and do XML normalization.

=item L<PYX::XMLSchema::List>

Processing PYX data or file and print list of XML schemas.

=item L<SGML::PYX>

Convertor between SGML and PYX.

=item L<Tags::Output::PYX>

Tags output to PYX format.

=item L<XML::MinWriter>

Perl extension for writing XML in PYX format.

=item L<XML::PYX>

PYX parser based on XML::Parser.

=back

=head1 REPOSITORY

L<https://github.com/michal-josef-spacek/Task-PYX>

=head1 AUTHOR

Michal Josef Špaček L<mailto:skim@cpan.org>

L<http://skim.cz>

=head1 LICENSE AND COPYRIGHT

© 2015-2023 Michal Josef Špaček

Artistic License

BSD 2-Clause License

=head1 VERSION

0.11

=cut


Makefile.PL
use lib '.';
use strict;
use warnings;

use inc::Module::Install;

# Definition.
abstract 'Install the PYX modules.';
author 'Michal Josef Spacek <skim@cpan.org>';
author_requires 'English' => 0;
author_requires 'File::Object' => 0.08;
author_requires 'Test::More' => 0;
author_requires 'Test::NoWarnings' => 0;
author_requires 'Test::Pod' => 0;
author_requires 'Test::Pod::Coverage' => 0;
license 'bsd';
name 'Task-PYX';
readme_from 'PYX.pm';
recursive_author_tests('xt');
requires 'App::PYX2XML' => 0.06;
requires 'PYX' => 0.10;
requires 'PYX::GraphViz' => 0.06;
requires 'PYX::Hist' => 0.08;
requires 'PYX::SGML::Raw' => 0.05;
requires 'PYX::SGML::Tags' => 0.10;
requires 'PYX::Sort' => 0.05;
requires 'PYX::Stack' => 0.06;
requires 'PYX::Utils' => 0.07;
requires 'PYX::XMLNorm' => 0.05;
requires 'PYX::XMLSchema::List' => 0.06;
requires 'SGML::PYX' => 0.07;
requires 'Tags::Output::PYX' => 0.05;
requires 'XML::MinWriter' => 0.08;
requires 'XML::PYX' => 0.07;
requires 'perl' => '5.6.0';
resources 'bugtracker' => 'https://github.com/michal-josef-spacek/Task-PYX/issues';
resources 'homepage' => 'https://github.com/michal-josef-spacek/Task-PYX';
resources 'repository' => 'git://github.com/michal-josef-spacek/Task-PYX';
test_requires 'Test::More' => 0;
test_requires 'Test::NoWarnings' => 0;
tests_recursive;
version '0.11';

# Run.
WriteAll();
